PHP ext version 3.13.0 - Build Warning

Building against PHP 7.4

```
p74-php-pecl-protobuf-3.13.0/NTS/names.c  -fPIC -DPIC -o .libs/names.o
/dev/shm/BUILD/php74-php-pecl-protobuf-3.13.0/NTS/message.c: In function 'Message_write_property':
/dev/shm/BUILD/php74-php-pecl-protobuf-3.13.0/NTS/message.c:269:3: warning: 'return' with no value, in function returning non-void [-Wreturn-type]
  269 |   return;
      |   ^~~~~~
/dev/shm/BUILD/php74-php-pecl-protobuf-3.13.0/NTS/message.c:245:25: note: declared here
  245 | static PROTO_RETURN_VAL Message_write_property(
      |                         ^~~~~~~~~~~~~~~~~~~~~~
/dev/shm/BUILD/php74-php-pecl-protobuf-3.13.0/NTS/message.c:276:3: warning: 'return' with no value, in function returning non-void [-Wreturn-type]
  276 |   return;
      |   ^~~~~~
/dev/shm/BUILD/php74-php-pecl-protobuf-3.13.0/NTS/message.c:245:25: note: declared here
  245 | static PROTO_RETURN_VAL Message_write_property(
      |                         ^~~~~~~~~~~~~~~~~~~~~~

```

--- a/php/ext/google/protobuf/message.c
+++ b/php/ext/google/protobuf/message.c
@@ -265,14 +265,14 @@ static PROTO_RETURN_VAL Message_write_property(
   }
 
   upb_msg_set(intern->msg, f, msgval, arena);
-#if PHP_VERSION_ID < 704000
+#if PHP_VERSION_ID < 70400
   return;
 #else
   return val;
 #endif
 
 error:
-#if PHP_VERSION_ID < 704000
+#if PHP_VERSION_ID < 70400
   return;
 #else
   return &EG(error_zval);

